clients/windows/src/input/mod.rs

```diff
@@ -5,5 +5,4 @@ pub mod message;
 pub mod mouse;
 pub mod pointer;
 
-pub use message::Message;
```

Comment:
I checked out the parent module and other submodules and didn't see anything taking advantage of this. It gets imported as message::{Message,...